ResearchKit/Consent/ORKConsentSignature.h

```diff
@@ -90,6 +90,13 @@ ORK_CLASS_AVAILABLE
                                     dateFormatString:(nullable NSString *)dateFormatString
                                           identifier:(NSString *)identifier;
 
+/**
+ Returns a boolean to indicate family name order.
+ 
+ Use this factory method to indicate family name first in the languages with a prefix @"zh", @"ko" and @"ja".
+ */
++ (BOOL)currentLocalePresentsFamilyNameFirst;
+
```

Comment:
Yeah, I agree with you. At first, I just thought it's all related to ORKConsentSignature. To put the method in ORKHelper is much better.